exporter/awsemfexporter/metric_translator.go

```diff
@@ -118,6 +134,268 @@ func (dps DoubleHistogramDataPointSlice) At(i int) DataPoint {
 	return dps.DoubleHistogramDataPointSlice.At(i)
 }
 
+
+// Retrieve namespace for given set of metrics from user config 
```

Comment:
Please start docstrings with function name:
```suggestion
// getNamespace retrieves namespace for given set of metrics from user config 
```